### ISSUE TITLE
API endpoint to check if a tweet is collected

### ISSUE THREAD

**Issue by yukiisbored** [loklak/loklak_server#1353]
- Issue type: Feature request 

An API endpoint that checks if a tweet is collected is helpful for those who want a tweet to be scanned and archived into Loklak's database in the same way on how the "Wayback Machine" works.

The API can look something familiar like below:
  * `/api/show.json?tweet=https://twitter.com/ArtisticBondage/status/885041783079661569`
  * `/api/show.json?tweet=885041783079661569`
  * `/api/show.json?user=https://twitter.com/ArtisticBondage`
  * `/api/show.json?user=ArtisticBondage`

The API does the following:
  * If it's a tweet, check if it exists or not
    * If it exists, show it.
    * If not, Scrap the tweet and add it to the db, return the same thing if it exists
  * If it's a user, check if we have anything by the user or not
    * If it exists, check how many tweets are archived from that user and archive the tweets that aren't archived yet and return what we know about the user and possibly the tweets (the follower count, etc)
    * If it doesn't exist, archive the tweets and the info about the user, return the same thing if it exists.

Possible use cases:
  * Archiving
  * Passive-way to receive new tweets

**Comment by vibhcool** [comment]
- Archiving is nice idea. It can be implemented by improving tweets dump implementation.

- Run the api endpoint, if the tweet related to info exists, exit, else collect the tweet?

**Comment by AnshulMalik** [comment]
I'd like to give it a try :)

**Comment by yukiisbored** [comment]
@AnshulMalik I've invited you to this org so I can assign you to this issue. Can you approve the invitation?

Thanks

**Comment by yukiisbored** [comment]
@vibhcool Yes, that's the idea. We can make Loklak into an archiving utility for tweets.

**Comment by AnshulMalik** [comment]
@yukiisbored I am in :)

I'll soon create a PR of my first attempt, then we can iterate over to build the feature.

**Comment by vibhcool** [comment]
@yukiisbored , but this feature exists in Loklak Server. dumps can be downloaded from `/dump.html` 😅

**Comment by yukiisbored** [comment]
@vibhcool But I can't see what those dumps contain and I can't see if they actually have the tweet I want nor I can request to archive a tweet ASAP.
